initfiles/componentfiles/configxml/esp.xsd.in

```diff
@@ -907,6 +916,13 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
+            <xs:attribute name="urlAlias" type="xs:string" use="optional">
+                <xs:annotation>
+                    <xs:appinfo>
+                        <tooltip>The URL alias to access this ESP.</tooltip>
```

Comment:
Reword with Jim's help

Reply:
The URL alias for this ESP. This can be used to detect Cross-Origin Resource Sharing (CORS) access.